examples/benchmarks/fabric-entertainment-curator/README.md

```
@@ -0,0 +1,136 @@
+### With LLM judge + real Memanto backend
+
+```bash
+# Get free Moorcheh API key at https://moorcheh.ai/
+export MOORCHEH_API_KEY=<your-key>
+export OPENAI_API_KEY=sk-<your-key>
+python run_benchmark.py
+```
+
+## File Structure
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)